operations/System_Recovery/PBS_Service_Recovery.md

```diff
@@ -0,0 +1,41 @@
+# PBS Service Recovery
+
+The following covers restoring PBS data.
+
+## Prerequisites
+
+The system is fully installed and has transitioned off of the LiveCD.
+
+All activities required for site maintenance are complete.
+
+A backup or export of the data already exists.
+
+The latest CSM documentation has been installed on the master nodes. See [Check for Latest Documentation](../../update_product_stream/index.md#check-for-latest-documentation).
```

Comment:
The CSM doc install requirement doesn't seem applicable for the PBS procedure, unless I'm overlooking something. 
```suggestion
- The system is fully installed and has transitioned off of the LiveCD.
- All activities required for site maintenance are complete.
- A backup or export of the data already exists.
```